### PR TITLE
feat: add db.sqlite to gitignore

### PR DESCRIPTION
**Pull request by c-ehrlich**
# Add db.sqlite to gitignore

- [x] I reviewed linter warnings + errors, resolved formatting, types and other issues related to my work
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

_[Short summary/description of story/feature/bugfix]_

As mentioned in https://github.com/t3-oss/create-t3-app/issues/200, the `db.sqlite` file created by the default prisma config was not included in .gitignore. 

I believe it's better to include it - Because we're not automatically creating a 'project initialized commit', it's easy to overlook the db file among all the other files that exist in version history after you initialize the app, and accidentally commit it.

If desired this could be changed to only include that line in the .gitignore if prisma is selected during initialization, but this way is much simpler and personally I think don't think it's an issue to just include it in .gitignore for everyone.

---

💯

### CODE REVIEW

**Comment by nexxeln** [comment]
Thank you

File: template/base/_gitignore [hunk 1]
```diff
@@ -8,6 +8,9 @@
 # testing
 /coverage
 
+# database
+db.sqlite
```
Comment (nexxeln) [inline template/base/_gitignore:12]: I think `db.sqlite` exists inside the prisma folder. So please change the path to `/prisma/db.sqlite`

**Review by nexxeln — CHANGES_REQUESTED** [review]

File: template/base/_gitignore (hunk 1 above)
Comment (c-ehrlich) [reply template/base/_gitignore]: It works this way (see screenshots) but I think you're right that being specific about the path is better. Will change.
 
Screenshot 2022-07-16 at 12 43 09
Screenshot 2022-07-16 at 12 42 55

**Review by nexxeln — APPROVED** [review]

**Comment by nexxeln** [comment]
Cool thanks!

**Pull request merged**